tests/prim/test_py_implementations.py

```diff
@@ -396,6 +400,30 @@ def test_prim_dot():
     assert (res == ref).all()
 
 
+def test_prim_scalar_bit_lshift():
+    ref = 3 << 2
+    res = scalar_bit_lshift(3, 2)
+    assert ref == res == 12
+
+
+def test_prim_scalar_bit_rshift():
+    ref = 12 >> 2
+    res = scalar_bit_rshift(12, 2)
+    assert ref == res == 3
+
+
+def test_prim_scalar_bit_or():
+    ref = 5 | 2
+    res = scalar_bit_or(5, 2)
+    assert ref == res == 7
+
+
+def test_prim_scalar_bit_xor():
+    ref = 10 ^ 8
+    res = scalar_bit_xor(10, 8)
+    assert ref == res == 2
+
+
```

Comment:
The coverage results are in, and it seems you forgot `test_prim_scalar_bit_and` :(